CoreVulkanDisplay/src/au/gov/asd/tac/constellation/visual/vulkan/renderables/CVKHitTester.java

```diff
@@ -328,10 +387,25 @@ public int DisplayUpdate() {
         if (!notificationQueues.isEmpty()) {
             final int x = hitTestRequest.getX();
             final int y = hitTestRequest.getY();
+            int redPixel = 0;
```

Comment:
Read this into a long in case the source data is unsigned.  It may not be required but you'd need to do a fair bit of digging to be sure, easier to just upsize the container.